bitwarden_license/src/Commercial.Infrastructure.EntityFramework/SecretsManager/Repositories/SecretRepository.cs

```diff
@@ -313,6 +313,34 @@ public async Task EmptyTrash(DateTime currentDate, uint deleteAfterThisNumberOfD
         await dbContext.SaveChangesAsync();
     }
 
+    public async Task MoveSecretsAsync(IEnumerable<Core.SecretsManager.Entities.Secret> secrets, IEnumerable<Guid> projectIds)
+    {
+        using var scope = ServiceScopeFactory.CreateScope();
+        var dbContext = GetDatabaseContext(scope);
+
+        var projectSecretsSet = dbContext.Set<Dictionary<string, object>>("ProjectSecret");
+
+        using var transaction = await dbContext.Database.BeginTransactionAsync();
```

Comment:
```suggestion
        await using var transaction = await dbContext.Database.BeginTransactionAsync();
```
My IDE is suggesting this since it implements `IAsyncDisposable`

Reply:
Good catch!